python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/router_bgp.schema.yml

```diff
@@ -185,6 +190,9 @@ keys:
               description: |-
                 BGP AS <1-4294967295> or AS number in asdot notation "<1-65535>.<0-65535>".
                 For asdot notation in YAML inputs, the value must be put in quotes, to prevent it from being interpreted as a float number.
+      send_community:
+        type: str
+        description: "`all` or a combination of `standard`, `extended` and `large`."
```

Comment:
I think we can keep vali values here:

```
large
extended
standard
extended large
standard large
standard extended
standard extended large
```

Reply:
Agreed, let's confirm once @ClausHolbechArista @gmuloc ?

Reply:
I can see the other areas with send_community also has `link-bandwidth (w/options)`. This may rely on hardware support, so check again with Fastcli -G.

Reply:
Seems like `link-bandwidth` extended community is only supported towards specific `neighbor` or `peer-group` and is not supported as an option for a `default neighbor`

Reply:
@ClausHolbechArista,  as @alexeygorbunov mentioned we have `link-bandwidth divide equal/ratio` and `link-bandwidth aggregate <0.0-4294967295.0>`  for non-default neighbors (checked with Fastcli -G)
shall we keep all the valid combinations in `valid-values`?

Reply:
Since we are just using the same model as elsewhere, and this PR is not about per-neighbor send-community, I don't see why we need to change anything here.
Whether to change it in general is a broader discussion with a huge fallout of work for very limited gain - and this is a rarely used config any way.